tests/basic_relay.go

```diff
@@ -96,8 +96,9 @@ func BasicRelay(network interfaces.LocalNetwork) {
 	)
 
 	relayerConfig := config.Config{
-		LogLevel:          logging.Info.LowerString(),
-		NetworkID:         peers.LocalNetworkID,
+		LogLevel: logging.Debug.LowerString(),
+		// TODO: There's currently a bug in ANR v1.7.4-rc.0 that specifies the network ID as 0. We should change this back to peers.LocalNetworkID once fixed.
+		NetworkID:         0,
```

Comment:
can you update `peer.LocalNetworkID` to use `github.com/ava-labs/avalanche-network-runner/utils/constants.DefaultNetworkID`?